app/models/penalties/PenaltyDetails.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.penalties
+
+import play.api.libs.json.{Format, Json}
+
+case class PenaltyDetails(LPPDetails : Seq[LPPDetails])
+
+object PenaltyDetails {
+
+  implicit val format: Format[PenaltyDetails] = Json.format[PenaltyDetails]
```

Comment:
Can you change this to be a JSON reads instead of a format. We don't need to write this model to JSON, and it will save you testing it